docs/user/openstack/README.md

```diff
@@ -287,26 +292,25 @@ Result:
 }
 ```
 
-Another useful thing to check is that the ignition configurations are
-only available from within the deployment. The following command should
-only succeed from a node in the OpenShift cluster:
+Another useful thing to check is that the ignition configurations are only available from within the deployment. The following command should only succeed from a node in the OpenShift cluster:
 
-* `curl https://<loadbalancer ip>:22623/config/master --insecure`
+```sh
+curl https://<loadbalancer ip>:22623/config/master --insecure
+```
 
 Now that the DNS and load balancer has been moved, we can take down the existing
-api VM:
+API VM:
 
-* `openstack server delete <cluster name>-api`
+```sh
+openstack server delete <cluster name>-api
+```
 
 ## Disambiguating the External Network
 
-The installer assumes that the name of the external network is unique.  In case
-there is more than one network with the same name as the desired external
-network, it’s possible to provide a UUID to specify which network should be
-used.
+The installer assumes that the name of the external network is unique. In case there is more than one network with the same name as the desired external network, it’s possible to provide a UUID to specify which network should be used.
 
-```
-$ env TF_VAR_openstack_external_network_id="6a32627e-d98d-40d8-9324-5da7cf1452fc" \
+```sh
+env TF_VAR_openstack_external_network_id="6a32627e-d98d-40d8-9324-5da7cf1452fc" \
```

Comment:
The users should not be asked to set these terraform specific envs. These should not end up in our public user documentation.

Reply:
cc @crawford

Reply:
This is a workaround for an edge case

Reply:
If we move it to a document exclusively for workarounds, can we still have it documented

Reply:
otherwise, we will file a bug report